File: example_problems/hello/submissions/run_time_error/test-sigfpe.java
/*
 * This should fail with RUN-ERROR due to integer division by zero.
 *
 * @EXPECTED_RESULTS@: RUN-ERROR
 */

import java.io.*;

class Main {
    public static void main(String[] args) throws Exception {
		int a = 0;
		int b = 10 / a;
		System.out.println(b);
    }
}

File: example_problems/hello/submissions/run_time_error/test-sigfpe.pas
(*
 * This should fail with RUN-ERROR due to a segmentation fault.
 *
 * @EXPECTED_RESULTS@: RUN-ERROR
 *)

program sigfpe(input, output);

var
   a,b : integer;

begin
   a := 1;
   b := 0;
   a := a div b;
end.
